src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -261,3 +278,95 @@ To use the EKCO add-on to remove a node:
 
 1. Remove the node from the cluster.
+
+## Troubleshoot Node Removal
+
+This section includes information about troubleshooting issues with node removal in Rook Ceph clusters.
+
+### Rook Ceph v1.0.4 is Unhealthy with Mon Pod Pending
+
+#### Symptom
+
+After you remove a node from a Rook Ceph v1.0.4 cluster and you run `kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status`, you see that Ceph is in an unhealthy state where a Ceph monitor (mon) is down. 
+
+For example:
+
+```
+health: HEALTH_WARN
+        1/3 mons down, quorum a,c
+```
+
+When you run `kubectl -n rook-ceph get pod -l app=rook-ceph-mon`, you see that the mon pod is in a Pending state. 
+
+For example:
+
+```
+NAME             READY  STATUS   RESTARTS  AGE
+rook-ceph-mon-a  1/1    Running  0         20m
+rook-ceph-mon-b  0/1    Pending  0         9m45s
+rook-ceph-mon-c  1/1    Running  0         13m
+```
+
+#### Cause
+
+This is caused by an issue in Rook Ceph v1.0.4 where the rook-ceph-mon-endpoints ConfigMap still maps a node that was removed.
+
+#### Solution
+
+To resolve this issue, you must return the Ceph cluster to a healthy state and upgrade to Rook Ceph v1.4 or later.
+
+To return Ceph to a healthy state so that you can upgrade, manually delete the mapping to the removed node from the rook-ceph-mon-endpoints ConfigMap then rescale the operator.
+
+To return Ceph to a healthy state and upgrade:
+
+1. Stop the Rook Ceph operator:
+
+   ```
+   kubectl -n rook-ceph scale --replicas=0 deployment.apps/rook-ceph-operator
+   ```
+
+1. Edit the rook-ceph-mon-endpoints ConfigMap to delete the removed node from the `mapping`:
+
+   ```
+   kubect -n rook-ceph edit configmaps rook-ceph-mon-endpoints
+   ```
+
+  _**Warning**_: Ensure that you remove the correct rook-ceph-mon-endpoint from the `mapping` field in the ConfigMap. Removing the wrong rook-ceph-mon-endpoint can cause unexpected behavior, including data loss.
+
+1. Find the name of the Pending mon pod:
+
+   ```
+   kubectl -n rook-ceph get pod -l app=rook-ceph-mon
+   ```
+
+1. Delete the Pending mon pod:
+
+   ```
+   kubectl -n rook-ceph delete pod MON_POD_NAME
+   ```
+   Replace `MON_POD_NAME` with the name of the mon pod that is in a Pending state from the previous step. 
+
+1. Rescale the operator:
+
+   ```
+   kubectl -n rook-ceph scale --replicas=1 deployment.apps/rook-ceph-operator
+   ```
+
+1. Verify that all mon pods are running:
+
+   ```
+   kubectl -n rook-ceph get pod -l app=rook-ceph-mon
+   ```
+   The output of this command shows that each mon pod has a `Status` of `Running`.
+
+1. Verify that Ceph is in a healthy state:
+
+   ```
+   kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status
+   ```
+
+   The output of this command shows `health: HEALTH_OK` if Ceph is in a healthy state.
+
+1. After confirming that Ceph is in a healthy state, upgrade Rook Ceph to v1.4 or later before attempting to managing nodes in the cluster.
```

Comment:
I added in this last step to explain that they really need to upgrade in order to address this issue.

Reply:
Should we say check if you can upgrade the Rook version used?
Not sure if we can say just upgrade and then if they are using a vendor app that does not work with the upper version?

Reply:
We state elsewhere in the kURL documentation that our requirement is to use v1.4 or later, so this should not be a concern.